addons/event/models/event.py

```diff
@@ -402,3 +405,12 @@ def action_send_badge_email(self):
             'target': 'new',
             'context': ctx,
         }
+
+
+class MailComposeMessage(models.Model):
+    _inherit = 'mail.compose.message'
+
+    @api.multi
+    def send_mail(self):
+        self.env['event.registration'].browse(self.env.context.get('active_id')).event_id.write({'rescheduled_alert': False})
```

Comment:
first call super method.
check active_model is a event.registration. otherwise it will be break...